Incorrect docs for RET504

I am looking at the docs for [RET504 (unnecessary-assign)](https://beta.ruff.rs/docs/rules/unnecessary-assign/). The following code is given there as an example:

```python
# test.py
def foo():
    bar = 1
    # some code that not using `bar`
    print('test')
    return bar
```

When I run `ruff check --select ALL test.py` on this code I get:
```shell
warning: `one-blank-line-before-class` (D203) and `no-blank-line-before-class` (D211) are incompatible. Ignoring `one-blank-line-before-class`.
warning: `multi-line-summary-first-line` (D212) and `multi-line-summary-second-line` (D213) are incompatible. Ignoring `multi-line-summary-second-line`.
test.py:1:1: D100 Missing docstring in public module
test.py:1:5: ANN201 Missing return type annotation for public function `foo`
test.py:1:5: D103 Missing docstring in public function
test.py:4:5: T201 `print` found
Found 4 errors.
```

There is no RET504 error. Is the example incorrect or am I missing something?

--- a/crates/ruff/src/rules/flake8_return/rules.rs
+++ b/crates/ruff/src/rules/flake8_return/rules.rs
@@ -150,16 +150,12 @@ impl AlwaysAutofixableViolation for ImplicitReturn {
 /// ```python
 /// def foo():
 ///     bar = 1
-///     # some code that not using `bar`
-///     print('test')
 ///     return bar
 /// ```
 ///
 /// Use instead:
 /// ```python
 /// def foo():
-///     # some code that not using `bar`
-///     print('test')
 ///     return 1
 /// ```
 #[violation]

